module/VuFindSearch/src/VuFindSearch/Command/TermsCommand.php

```diff
@@ -111,7 +111,7 @@ public function getArguments(): array
      *
      * @return string
      */
-    public function getField(): string
+    public function getField(): ?string
```

Comment:
The doc comments above the getters should be adjusted to have consistent types.

Reply:
fixed